lib/common/common.go

```diff
@@ -51,14 +51,10 @@ func ParseDockerURL(arg string) *types.ParsedDockerURL {
 	}
 	indexURL, imageName := SplitReposName(taglessRemote)
 
-	if indexURL == "" && !strings.Contains(imageName, "/") {
+	if indexURL == defaultIndexURL && !strings.Contains(imageName, "/") {
```

Comment:
Yep, the Docker client considers images referenced only by a name (e.g. `busybox`, `ubuntu` instead of `coreos/etcd` or `localhost/whatever`) as valid, and, in that case, it adds the `library/` prefix because that's how they're stored in the docker.io registry.